python/paddle/fluid/contrib/slim/tests/slim_int8_mkldnn_post_training_quantization.md

```diff
@@ -0,0 +1,67 @@
+# SLIM Post-training quantization (INT8 MKL-DNN)
+
+This document describes how to use [Paddle Slim](https://github.com/PaddlePaddle/FluidDoc/blob/develop/doc/fluid/advanced_usage/paddle_slim/paddle_slim.md) to convert a FP32 ProgramDesc with FP32 weights to an INT8 ProgramDesc with FP32 weights on GoogleNet, MobileNet-V1, MobileNet-V2, ResNet-101, ResNet-50, VGG16 and VGG19. Paddle Slim MKL-DNN post-training quantization strategy accuracy results on above 7 models are as follows:
+
+## Accuracy benchmark
+
+>**I. Top-1 Accuracy on Intel(R) Xeon(R) Gold 8280**
+
+| Model        | Dataset                        | FP32 Accuracy   | INT8 Accuracy   | Accuracy Diff   |
+| :----------: | :----------------------------: | :-------------: | :------------:  | :--------------:|
+| GoogleNet    | ILSVRC2012 Validation dataset  |  70.50%         |  70.20%         |  0.30%          |
+| MobileNet-V1 | ILSVRC2012 Validation dataset  |  70.78%         |  70.36%         |  0.42%          |
+| MobileNet-V2 | ILSVRC2012 Validation dataset  |  71.90%         |  71.57%         |  0.33%          |
+| ResNet-101   | ILSVRC2012 Validation dataset  |  77.50%         |  77.53%         | -0.03%          |
+| ResNet-50    | ILSVRC2012 Validation dataset  |  76.63%         |  76.48%         |  0.15%          |
+| VGG16        | ILSVRC2012 Validation dataset  |  72.08%         |  72.01%         |  0.07%          |
+| VGG19        | ILSVRC2012 Validation dataset  |  72.56%         |  72.56%         |  0.00%          |
+
+Notes:
+
+* MKL-DNN and MKL are required.
+
+## Instructions to reproduce the above accuracy benchmark
+
+### 0. Install PaddlePaddle
+
+Follow PaddlePaddle [installation instruction](http://www.paddlepaddle.org/documentation/docs/en/1.4/beginners_guide/install/index_en.html) to install PaddlePaddle. If you [build from source](https://github.com/PaddlePaddle/FluidDoc/blob/develop/doc/fluid/beginners_guide/install/compile/compile_Ubuntu_en.md), please use the following cmake arguments.
+
+```bash
+cmake .. -DCMAKE_BUILD_TYPE=Release -DWITH_GPU=OFF -DWITH_MKL=ON -DWITH_MKLDNN=ON  -DWITH_TESTING=ON  -WITH_FLUID_ONLY=ON  -DWITH_INFERENCE_API_TEST=ON -DON_INFER=ON -DWITH_SLIM_MKLDNN_FULL_TEST=ON
```

Comment:
- `-WITH_FLUID_ONLY=ON` is removed after release1.3. please remove it.
- @bingyanghuang 
please update the cmake option in `int8_mkldnn_quantization.md` at the same time. I see `-DWITH_MKLDNN=ON` is missing in `int8_mkldnn_quantization.md`

Reply:
done